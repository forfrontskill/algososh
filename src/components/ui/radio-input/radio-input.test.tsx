import renderer from 'react-test-renderer';
import { RadioInput } from './radio-input';

jest.mock('nanoid', () => ({
    nanoid: jest.fn(()=>'test')
  }))

describe('Test RadioInput component', () => {

    it('RadioInput rendering', () => {
        const tree = renderer
            .create(<RadioInput id='key' label='test' htmlFor='test'/>)
            .toJSON();
        expect(tree).toMatchSnapshot();
    });
});