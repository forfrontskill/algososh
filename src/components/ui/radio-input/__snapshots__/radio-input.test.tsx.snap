// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test RadioInput component RadioInput rendering 1`] = `
<div
  className="content "
>
  <input
    className="input"
    htmlFor="test"
    id="key"
    type="radio"
  />
  <label
    className="text text_type_button label"
  >
    test
  </label>
</div>
`;
